<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>i2c</class>
 <widget class="QWidget" name="i2c">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>i2c</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>220</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbxPort"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_frequency_7">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbxFrequency">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_mode_6">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbxMode">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblCS">
        <property name="text">
         <string>Chip select</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbxCS">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Erase before write</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbnEraseBeforeWriteYes">
          <property name="text">
           <string>Yes</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbnEraseBeforeWriteNo">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_page_size_6">
        <property name="text">
         <string>Page size</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cbxPageSize">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_total_size_5">
        <property name="text">
         <string>Total size</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cbxTotalSize">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>601</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Actions</string>
   </property>
   <widget class="QPushButton" name="btnReadToFile">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>63</y>
      <width>180</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Read device into file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetExtendedID">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Extended ID</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtDeviceID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>53</y>
      <width>180</width>
      <height>130</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWriteFromFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>63</y>
      <width>180</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Write file into device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnErase">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>106</y>
      <width>180</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase all sectors</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
